src/ui/components/EditableCard.stories.tsx

```diff
@@ -0,0 +1,50 @@
+import { Meta, StoryObj } from '@storybook/react';
+import { EditableCard } from '@/ui/components/EditableCard';
+
+const meta: Meta<typeof EditableCard> = {
+  /* 👇 The title prop is optional.
+   * See https://storybook.js.org/docs/react/configure/overview#configure-story-loading
+   * to learn how to generate automatic titles
+   */
+  title: 'components/EditableCard',
+  component: EditableCard,
+  args: {
+    iconName: 'angle-down',
+    title: 'Card title',
+    description: 'This is the description',
+    value: 'This is the value',
+    actions: [],
+  },
+};
+
+export default meta;
+type Story = StoryObj<typeof EditableCard>;
+
+export const Primary: Story = {
+  args: {
+    title: 'Card title2',
+  },
+};
+
+export const Profile: Story = {
```

Comment:
This is the new StoryBook format, which I would hope we can use. But at the moment, I have not found a way to teach it to work with `args` correctly. The problem is that TypeScript can't tell us if args are missing – the story will silently crash. This is why I'm still using the old format …